tools/spades_header_fixer/.shed.yml

```diff
@@ -0,0 +1,4 @@
+categories: [Sequence Analysis]
+description: Fixes Spades Fasta ids
+name: spades_fasta_header_fixer
+owner: nml
```

Comment:
Can you add  **remote_repository_url** and **homepage_url** values as well please. Just use values https://github.com/phac-nml/galaxy_tools

Reply:
Done, fixed in commit 2946ebc